plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -93,7 +94,7 @@ function CoverMenu:updateItems(select_number)
             -- reset focus manager accordingly
             self.selected = { x = 1, y = select_number }
             -- set focus to requested menu item
-            self.item_group[select_number]:onFocus()
+            self:getFocusItem():handleEvent(Event:new("Focus"))
```

Comment:
@poire-z Oh sorry, ignore https://github.com/koreader/koreader/pull/3745#issuecomment-374735641 It's fixed here already.

Reply:
oh, good :)